components/browser/thumbnails/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <dimen name="mozac_browser_thumbnails_size_default">102dp</dimen>
+
+    <dimen name="mozac_browser_thumbnails_maximum_size" tools:ignore="PxUsage">2153px</dimen>
```

Comment:
Where did this value come from? I wonder if there is a better DP value we can choose here.

Future: if we could ask the app to provide the app with this attribute based on the thumbnail visible and we can store an image size that is optimized for what is visible.

Reply:
I took the max of the height/width bounds I was seeing from the Bitmap coming from engineView.captureThumbnails. I expect we will need to tweak all the DesiredSizes numbers.

Reply:
With pixel values, those are absolute numbers. With browser icons I think the limitation is that favicons have low image quality so we keep a hard pixel value on the image. With thumbnails though, I imagine the limitation is what the displayed view size is which is relative to your device.

> I took the max of the height/width bounds I was seeing from the Bitmap coming from engineView.captureThumbnails.

This value will change if you test it on a device with a different display.

Reply:
Will explore this in a followup.